setup.py

```diff
@@ -11,7 +11,7 @@
 
 setup(
     name='masquerade',
-    version='1.1.7',
```

Comment:
**question**: do you want the version bump in here?

Reply:
Oops. Wrong target branch. I switched it to dev. And TBH - I can't remember if the version bump is automated or not. I'll check on it when I promote from dev to main.